### PR TITLE
fix netqtop python3 compatible

### PR DESCRIPTION
**Pull request by Yolandajn**
fixed python3 compatible issue referred in #3139 and [pr#3048](https://github.com/iovisor/bcc/pull/3048#issuecomment-710483485)

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, test this please]

File: tools/netqtop.py [hunk 1]
```diff
@@ -28,7 +29,7 @@ def to_str(num):
     elif num > 1000:
         return str(round(num/1024.0, 2)) + 'K'
     else:
-        if type(num) == types.FloatType:
+        if isinstance(num, float):
```
Comment (yonghong-song) [inline tools/netqtop.py:32]: Since we removed using `types` module, we do not need to import it (line 9), right?

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

File: tools/netqtop.py (hunk 1 above)
Comment (Yolandajn) [reply tools/netqtop.py:31]: yes, thank you. I deleted this line.

**Pull request merged**